### PR TITLE
Credential

### PR DESCRIPTION
**Pull request by stephaniestroka**
Added a CredentialsSignatureCreatorTest that verifies whether a message can be signed
with a previously issued client credential.

### CODE REVIEW

File: signature/src/main/java/net/digitalid/core/signature/credentials/CredentialsSignature.java [hunk 1]
```diff
@@ -107,6 +107,7 @@ public void verifySignature() throws InvalidSignatureException, ExpiredSignature
         final @Nonnull BigInteger hash = getContentHash(getTime(), getSubject(), getObjectConverter(), getObject());
         
         if (getSU().getBitLength() > Parameters.RANDOM_EXPONENT.get()) {
+            System.out.println("SU: " + getSU() + ", bit length: " + getSU().getBitLength() + ", length of random exponent: " + Parameters.RANDOM_EXPONENT.get());
```
Comment (KasparEtter) [inline signature/src/main/java/net/digitalid/core/signature/credentials/CredentialsSignature.java]: Maybe rather log this than printing it to standard output? And maybe without SU itself?

File: signature/src/main/java/net/digitalid/core/signature/credentials/CredentialsSignatureCreator.java [hunk 2]
```diff
@@ -53,23 +53,14 @@
     public interface SubjectCredentialsSignatureCreator<OBJECT> {
 
         /**
-         * Addresses the signature to a certain subject.
+         * Addresses the signature as a subject.
          */
         @Pure
-        public @Nonnull LodgedCredentialsSignatureCreator<OBJECT> to(@Nonnull InternalIdentifier subject);
+        public @Nonnull CommitmentCredentialsSignatureCreator<OBJECT> as(@Nonnull InternalIdentifier subject);
```
Comment (KasparEtter) [inline signature/src/main/java/net/digitalid/core/signature/credentials/CredentialsSignatureCreator.java]: It's not an "as" (i.e. not as a given signer) but rather "about" (i.e. about a given subject), isn't it?

**Review by KasparEtter — CHANGES_REQUESTED** [review]

File: signature/src/main/java/net/digitalid/core/signature/credentials/CredentialsSignature.java (hunk 1 above)
Comment (stephaniestroka) [reply signature/src/main/java/net/digitalid/core/signature/credentials/CredentialsSignature.java]: I actually removed the whole line last week, but forgot to commit.

**Review by KasparEtter — DISMISSED** [review]

**Comment by KasparEtter** [comment]
You can now squash and merge. 🙂

**Review by KasparEtter — APPROVED** [review]

**Pull request merged**